package-lock.json

```diff
@@ -1050,17 +1050,17 @@
       "integrity": "sha512-x5HFsW+E/nQalGMw7hu+fvPqnBeBaIr0lWJ2SG0PPL2j+Pm9lYvCrsZJGIgauPIENx0v10INIyFjmSNUD/gSqQ=="
     },
     "@babel/plugin-external-helpers": {
-      "version": "7.8.3",
```

Comment:
Not sure what this entire diff is about, but since you haven't added any dependencies, please undo it.